troposphere/static/js/components/modals/api_token/APITokenCreate.jsx

```diff
@@ -0,0 +1,190 @@
+import React from "react";
+import BootstrapModalMixin from "components/mixins/BootstrapModalMixin";
+import actions from "actions";
+import {RaisedButton, CircularProgress} from "material-ui";
+import WarningIcon from "material-ui/svg-icons/alert/warning";
+import CopyButton from "components/common/ui/CopyButton";
+
+export default React.createClass({
+    mixins: [BootstrapModalMixin],
+    propTypes: {
+        user: React.PropTypes.number.isRequired
+    },
```

Comment:
In `getInitialState` we were assigning to "name" the "name" prop and letting in fall through to empty string if not defined `name: name || ""` . I removed the use of the "name" prop from Create modal as it was left over from when the Edit modal and Create modal were sharing code. Since separating these experiences into their own files we no longer need a "name" prop on the Create modal.